discovery-provider/src/models/models.py

```diff
@@ -178,8 +178,11 @@ def __repr__(self):
 class User(Base):
     __tablename__ = "users"
 
-    blockhash = Column(String, ForeignKey("blocks.blockhash"), nullable=False)
-    blocknumber = Column(Integer, ForeignKey("blocks.number"), nullable=False)
+    blockhash = Column(String, ForeignKey("blocks.blockhash"), nullable=True)
```

Comment:
just to clarify we are not gonna have a slots table like we do for blocks in POA? there'll be too much data anyway

Reply:
There will be a slots table I forgot to add here, but hareesh added in a followup task - https://github.com/AudiusProject/audius-protocol/pull/2815